hw2/05-form.js

```diff
@@ -1 +1,32 @@
 // Add your code here
+const fullName = document.getElementById("fullName");
+const email = document.getElementById("email");
+const registrationStatus = document.getElementById("registrationStatus");
+const programmingLanguages = document.getElementById("programmingLanguages");
+const operatingSystems = document.getElementById("operatingSystems");
+const fullStackWebDevelopment = document.getElementById(
+  "fullStackWebDevelopment"
+);
+const anythingElse = document.getElementById("anythingElse");
```

Comment:
Instead of querying the DOM repeatedly for the elements in the form, it would be easier to use form.elements. Read about it [here.](https://eloquentjavascript.net/2nd_edition/18_forms.html)

Reply:
That makes sense.